### PR TITLE
[SDFAB-829] Add support for session and application metering - 2

### PR DESCRIPTION
**Pull request by daniele-moro**
Adding meter configuration to the `p4rt-forwarding` test scenario.
The effectiveness of the meters is not verified in the test, we only verify the programming of meters is done correctly.
Indeed, the app and session meters are both pushed even if we don't have any application filtering rule.

### CODE REVIEW

**Comment by ccascone** [comment]
I'd be inclined to actually modify the existing `p4rt-forwarding` scenario instead of creating a new one. Programming meters in the UPF is pretty common. More scenarios mean longer execution times for CI.

File: scenarios/bin/p4rt-up4-flows [hunk 1]
```diff
@@ -68,6 +74,10 @@ def get_args():
                         help="Drop traffic in uplink direction")
     parser.add_argument("--drop-dl", type=bool, default=False,
                         help="Drop traffic in downlink direction")
+    parser.add_argument("--app-bw-meter", type=int, default=None,
+                        help="App meter max bitrate")
```
Comment (pierventre) [inline scenarios/bin/p4rt-up4-flows:78]: in b/s ? maybe is better to specify the unit ?

File: scenarios/bin/p4rt-up4-flows [hunk 2]
```diff
@@ -68,6 +74,10 @@ def get_args():
...
+    parser.add_argument("--sess-bw-meter", type=int, default=None,
+                        help="Session meter max bitrate")
```
Comment (pierventre) [inline scenarios/bin/p4rt-up4-flows:80]: same here

File: scenarios/bin/p4rt-up4-flows [hunk 3]
```diff
@@ -113,6 +123,20 @@ def add_entry(entry, action):
...
 
+def mod_meter(name, index, max_bw, action):
+    meter = sh.MeterEntry(name)
+    meter.index = index
+    if action == "program" or action == "dry":
+        # in case of "clear" don't specify bands to clear meters
+        meter.cir = 1
+        meter.cburst = 1
+        meter.pir = max_bw // 8
+        meter.pburst = int((max_bw / 8) * BURST_DURATION_MS * 0.001)
```
Comment (pierventre) [inline scenarios/bin/p4rt-up4-flows:134]: pir ?

File: scenarios/bin/p4rt-up4-flows [hunk 4]
```diff
@@ -113,6 +123,20 @@ def add_entry(entry, action):
             sys.exit(1)
     entries.append(entry)
 
+def mod_meter(name, index, max_bw, action):
```
Comment (pierventre) [inline scenarios/bin/p4rt-up4-flows:126]: and where do you generate the app meter ?

Comment (pierventre) [reply scenarios/bin/p4rt-up4-flows:126]: You are always setting cir and burst to 1

**Review by pierventre — APPROVED** [review]

**Comment by daniele-moro** [comment]
Abandoning, superseded by: #241

**Pull request closed without merging**